### PR TITLE
Rename job completion index annotation to not contain alpha

### PR DESCRIPTION
**Pull request by alculquicondor**
#### What type of PR is this?

/kind bug
/kind api-change

#### What this PR does / why we need it:

API conventions now discourage use of "alpha" or "beta" in the annotation name
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#label-selector-and-annotation-conventions

#### Which issue(s) this PR fixes:

Ref kubernetes/enhancements#2214

#### Does this PR introduce a user-facing change?

Feature not released yet

```release-note
NONE
```

### CODE REVIEW

File: api/openapi-spec/swagger.json [hunk 1]
```diff
@@ -11857,6 +11857,552 @@
       },
       "type": "object"
     },
+    "io.k8s.api.flowcontrol.v1alpha1.FlowDistinguisherMethod": {
```
Comment (liggitt) [inline api/openapi-spec/swagger.json:11860]: this looks spurious... regenerate?

**Review by liggitt — COMMENTED** [review]
one comment on generated file, lgtm otherwise

File: api/openapi-spec/swagger.json (hunk 1 above)
Comment (alculquicondor) [reply api/openapi-spec/swagger.json]: Uhm... still getting it. I had to manually delete the change

Comment (liggitt) [reply api/openapi-spec/swagger.json]: rebase on `master`, `make clean`, maybe?

Comment (alculquicondor) [reply api/openapi-spec/swagger.json]: it worked the 2nd time 🤷‍♂️

**Comment by liggitt** [comment]
/triage accepted
/lgtm
/approve

this was an oversight in the review... guidance is to not include alpha in annotation keys because of difficulties transitioning out of alpha

**Pull request merged**